### PR TITLE
Adding first pass at the SECURITY-INSIGHTS.yml

### PR DESCRIPTION
**Pull request by jkowall**
## Which problem is this PR solving?
- Adding first pass at SECURITY-INSIGHTS.yml file

## Description of the changes
- Adding file based on the project configuration to satisfy new requirements from CLOMonitor

## How was this change tested?
- No testing needed as this file is not used internally to the project

## Checklist
- [X] I have read https://github.com/jaegertracing/jaeger/blob/master/CONTRIBUTING_GUIDELINES.md
- [X] I have signed all commits
- [NOT NEEDED] I have added unit tests for the new functionality
- [NOT NEEDED] I have run lint and test steps successfully
  - for `jaeger`: `make lint test`
  - for `jaeger-ui`: `yarn lint` and `yarn test`

### CODE REVIEW

File: SECURITY-INSIGHTS.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  last-updated: '2023-10-20'
+  last-reviewed: '2023-10-20'
+  expiration-date: '2024-10-20T01:00:00.000Z'
+  project-url: https://github.com/jaegertracing/jaeger/
+  changelog: https://github.com/jaegertracing/jaeger/blob/main/CHANGELOG.md
+  license: https://github.com/jaegertracing/jaeger/blob/main/LICENSE
+project-lifecycle:
+  bug-fixes-only: false
+  core-maintainers:
+  - https://github.com/orgs/jaegertracing/teams/jaeger-maintainers
```
Comment (yurishkuro) [inline SECURITY-INSIGHTS.yml:12]: not sure this is visible, maybe link to README#maintainers

File: SECURITY-INSIGHTS.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+contribution-policy:
+  accepts-pull-requests: true
+  accepts-automated-pull-requests: true
+  contributing-policy: https://github.com/jaegertracing/jaeger/blob/main/CONTRIBUTING.md 
+  code-of-conduct: https://github.com/jaegertracing/jaeger/blob/main/CODE_OF_CONDUCT.md
+documentation:
+- https://www.jaegertracing.io/docs/
+distribution-points:
+- https://github.com/jaegertracing/jaeger/
+- https://hub.docker.com/r/jaegertracing/
```
Comment (yurishkuro) [inline SECURITY-INSIGHTS.yml:25]: also https://quay.io/organization/jaegertracing

File: SECURITY-INSIGHTS.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+vulnerability-reporting:
+  accepts-vulnerability-reports: true
+  security-policy: https://github.com/jaegertracing/jaeger/blob/main/SECURITY.md
+  email-contact: jaeger-tracing@googlegroups.com
+  comment: |
+    The first and best way to report a vulnerability is by using private security issues in GitHub or opening an issue on Github. We are also available on the CNCF Slack in the jaeger channel.
+dependencies:
+  third-party-packages: true
+  dependencies-lists:
+  - https://github.com/foo/packages.json
```
Comment (yurishkuro) [inline SECURITY-INSIGHTS.yml:51]: ?

Comment (jkowall) [reply SECURITY-INSIGHTS.yml]: Meant to remove it, but forgot. Handled on latest push.

File: SECURITY-INSIGHTS.yml (hunk 2 above)
Comment (jkowall) [reply SECURITY-INSIGHTS.yml:25]: Added, thanks!

File: SECURITY-INSIGHTS.yml (hunk 1 above)
Comment (jkowall) [reply SECURITY-INSIGHTS.yml]: Yes, you at least need to be authenticated, so that's better, updated.

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**